cmd/agent/entrypoint.go

```diff
@@ -61,6 +65,14 @@ func NewEntrypoint(logger *server.Logger, cfg *config.Config, reloader Reloader)
 		ep = &Entrypoint{
 			log:      logger,
 			reloader: reloader,
+			configReloadSuccess: promauto.NewGauge(prometheus.GaugeOpts{
+				Name: "agent_config_last_reload_successful",
+				Help: "Config loaded successfully.",
+			}),
+			configReloadSeconds: promauto.NewGauge(prometheus.GaugeOpts{
```

Comment:
Check with Goutham but I wonder if its better to have success/failure be a label so we can track the two separately, it does break some standing with prometheus but might be acceptable.  So would become `agent_config_last_reload_timestamp_seconds` with label like `result=success` or `result=failure`.

Reply:
I thought about, it's probably a good idea, let's do it